docs/configuration.md

```
@@ -22,6 +22,53 @@ Polylogue follows XDG Base Directory specification:
+Use `polylogue paths` to print the resolved paths for the current
+environment, or `polylogue paths --format json` for machine-readable
+output. The command also reports any detected bind mounts and
+non-canonical files at the archive root.
+
+### Bind Mounts and `/realm/data/captures/`
+
+Some deployments (such as the [sinnix](https://github.com/Sinity/sinnix)
+NixOS configuration) expose the archive at two path strings through a
+btrfs subvolume bind mount:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)